backend/curation/api/curation-api.yml

```diff
@@ -1457,6 +1459,14 @@ components:
       required: true
       schema:
         $ref: "#/components/schemas/dataset_version_id"
+    schema_version:
+      description: Filter to fetch latest version of dataset with the given CELLxGENE schema version
+      in: query
+      name: schema_version
+      required: false
+      schema:
+        type: string
+        example: "3.0.0"
```

Comment:
What's the behavior if the schema version is unavailable?

5.0.0 or 1.1.0 ?

Reply:
No schema match == No results.

But that's flexible. I'm thinking the downstream consumers that will want to use this will have the necessary context to pass a valid schema version. And it won't cause breaking changes if we move past a particular version and consumers don't update, since we'll have the previous versions stored to serve them.

Reply:
That's fine. I think it should be documented, because the other alternative is to return an error status code.